components/common/DataTable.vue

```diff
@@ -30,10 +30,10 @@ const css = useCssModule();
             <th
               v-for="header in headers"
               :key="header.text"
-              :class="{
-                [header.className]: true,
-                [css.header]: true,
-              }"
+              :class="[
+                ...(header.className ? [header.className] : []),
```

Comment:
I think `[header.className, css.header]` will still work here
I know undefined/null don't get assigned as classes.

Reply:
Let me try it to see if it works.

Reply:
Ok, it seems to work fine. thanks for the suggestion :)